### DISCUSSION TITLE
Nuke, maya and houdini

### DISCUSSION THREAD

**Discussion by ronnyascencio** [jonassorgenfrei/AssetBrowser#2] [Q&A]
Hello is this working with nuke 13 maya 2023 and houdini 19? all python 3 I would like to test this for personal work and keep organized my stuff

**Comment by jonassorgenfrei** [comment]
Hey @ronnyascencio,
the  current state of this project/repo is very WIP it is mainly just the basic construct of the system.
At the moment it only contains the Polyheaven conection, as i messed the Ui loading a little up with the latest comits I have to clean up some stuff in the next days/weeks to make it a real usefull/working tool again. 

But yeah to answer your question it is working with the latest versions of Nuke and Houdini right out of the box when you add the packages as well as the new maya. But for maya i don't have an autoload script yet, so you would have to manually add the widget. So everythinf python3 ready and yeah i don't actually plan to support python2 still with it but.

With that said, feel free to test it out to get a sneakpeak of the ui and let me know toughts/problems/ideas. 
Stay tuned over the next weeks i will add more functionality to make it a use full tool in daily work.

**Reply by ronnyascencio** [reply]
Thank you for your answer Because I am trying to install and its not working maybe im doing something wrong goin to try 

thank you and Ill let you know thank you for develop the tools its awesome!

**Reply by jonassorgenfrei** [reply]
You're welcome. What is not working? 
May be try it with Houdini using the package system first. This is aswell the main application where I develop this tool tho. 
Let me know what is not working then I will try to help you or I will take a look on the code to adjust it accordingly.
